lib/seahorn/BvOpSem2TrackingRawMemMgr.hh

```diff
@@ -225,15 +282,19 @@ public:
 
   MemValTy zeroedMemory() const;
 
+  MemValTy setMemory(unsigned int val) const;
```

Comment:
I know believe that this should be `initializedMemory` and `zeroedMemory` should be `zeroInitializedMemory`
Note that `zeroinitialized` is a keyword LLVM uses to mark a global memory region `zeroinitialized`, so the name will make sense to anyone familiar with llvm